base-requirements.txt

```diff
@@ -1,46 +1,47 @@
 dj-database-url==0.5.0
-django-pipeline==1.6.14
-django-sitetree==1.10.0
-Django==2.0.13
+django-pipeline==2.0.6
+django-sitetree@https://github.com/idlesign/django-sitetree/archive/a2d77e9c282fc4be60698a8421c42a494393e794.zip
+Django<=2.3.0,>2.1.0
 docutils==0.12
 Markdown==3.3.4
-cmarkgfm==0.4.2
+cmarkgfm==0.6.0
 Pillow==8.3.1
 psycopg2==2.8.6
-python3-openid==3.1.0
+python3-openid==3.2.0
+python-decouple==3.4
 # lxml used by BeautifulSoup.
 lxml==4.6.3
-cssselect==0.9.1
+cssselect==1.1.0
 feedparser==6.0.8
-beautifulsoup4==4.6.0
-icalendar==3.8.4
+beautifulsoup4==4.9.3
+icalendar==4.0.7
 chardet==4.0.0
 # TODO: We may drop 'django-imagekit' completely.
 django-imagekit==4.0.2
-git+https://github.com/django-haystack/django-haystack.git@802b0f6f4b3b99314453261876a32bac2bbec94f
+django-haystack==3.0
 elasticsearch>=5,<6
 # TODO: 0.14.0 only supports Django 1.8 and 1.11.
-django-tastypie==0.14.1
+django-tastypie==0.14.3
 
-pytz==2017.2
-python-dateutil==2.8.1
+pytz==2021.1
+python-dateutil==2.8.2
 
 requests[security]>=2.26.0
 
-django-honeypot==0.6.0
-django-markupfield==1.4.3
+django-honeypot==1.0.1
+django-markupfield==2.0.0
 django-markupfield-helpers==0.1.1
 
 django-allauth==0.41.0
 
-django-waffle==0.14
+django-waffle==2.2.1
 
-djangorestframework==3.8.2
-django-filter==1.1.0
-django-ordered-model==3.4.1
+djangorestframework==3.12.2
+django-filter==2.4.0
+django-ordered-model==3.4.3
 django-widget-tweaks==1.4.8
-django-countries==6.1.3
+django-countries==7.2.1
 xhtml2pdf==0.2.5
-django-easy-pdf==0.1.1
+django-easy-pdf@https://github.com/labcodes/django-easy-pdf/archive/60f2a70a056655d271a215ceded57f97ffaa4fea.zip
```

Comment:
@luzfcb why to use a custom fork instead of a regular version?

Reply:
Is only this change:
https://github.com/labcodes/django-easy-pdf/commit/60f2a70a056655d271a215ceded57f97ffaa4fea

Was required to replace 
`from django.utils.six import BytesIO`
by 
`from io import BytesIO `

because the `django.utils.six` is deprecated.

Anyway, django-easy-pdf seems unmaintained project

Reply:
that'd be preferred, unless the fork here is acceptable: https://pypi.org/project/django-easy-pdf3/ it seems that the only diff is _effectively_ the same change

Reply:
I switched to django-easy-pdf3